diff --git a/node_modules/@astrojs/vercel/dist/index.js b/node_modules/@astrojs/vercel/dist/index.js
index f3fb62cd0a9be9314860e540ad7cdbc9b057af59..78b920ef9472090aebeba5c5c10355dd266b9b59 100644
--- a/node_modules/@astrojs/vercel/dist/index.js
+++ b/node_modules/@astrojs/vercel/dist/index.js
@@ -1,4 +1,5 @@
-import { cpSync, existsSync, mkdirSync, readFileSync } from "node:fs";
+import fs from "node:fs";
+import { existsSync, mkdirSync, readFileSync } from "node:fs";
 import { basename } from "node:path";
 import { pathToFileURL } from "node:url";
 import { emptyDir, removeDir, writeJson } from "@astrojs/internal-helpers/fs";
@@ -131,7 +132,7 @@
                 "astro:build:done": async () => {
                   logger.info("Copying static files to .vercel/output/static");
                   const _staticDir = _buildOutput === "static" ? _config.outDir : _config.build.client;
-                  cpSync(_staticDir, new URL("./.vercel/output/static/", _config.root), {
+                  await fs.promises.cp(_staticDir, new URL("./.vercel/output/static/", _config.root), {
                     recursive: true
                   });
                 }
@@ -236,7 +237,7 @@
           });
           mkdirSync(new URL("./.vercel/output/server/", _config.root));
           if (_buildOutput !== "static") {
-            cpSync(_config.build.server, new URL("./.vercel/output/_functions/", _config.root), {
+            await fs.promises.cp(_config.build.server, new URL("./.vercel/output/_functions/", _config.root), {
               recursive: true
             });
           }
